lib/definitions/eep/TF-13/TF-13-07.json

```diff
@@ -53,39 +65,16 @@
                 "type": "number",
                 "read": true,
                 "write": true,
-                "states": "0:false; 1:true",
+                "states": {
+                  "0" : "false",
+                  "1" : "true"
+                },
                 "def": 0
             },
             "native": {}
         }
     },
     "case": [
-        {
-            "send": false,
-            "condition": {
-                "command": [
-                    {
-                        "value": 255
-                    }
-                ]
-            },
-            "datafield": [
-                {
-                    "data": "Level",
-                    "shortcut": "LVL",
-                    "bitoffs": "8",
-                    "bitsize": "8",
-                    "value": 100
-                },
-                {
-                    "data": "Dimming speed",
-                    "shortcut": "DS",
-                    "bitoffs": "16",
-                    "bitsize": "8",
-                    "value": 0
-                }
-            ]
-        },
```

Comment:
Why did you delete this part?

Reply:
It's not deleted. Just moved to the end, so that all 6 sends are together and the read part comes last. It's just cosmetic, easier to read.